packages/provider/wrapper.js

```diff
@@ -32,6 +32,43 @@ module.exports = {
       const originalSendAsync = provider.sendAsync.bind(provider);
       provider.sendAsync = this.send(originalSendAsync, preHook, postHook);
     }
+
+    // EIP-1193 provider wrapping
+    if (
+      typeof provider.request === "undefined" &&
+      typeof provider.send !== "undefined"
+    ) {
+      provider.request = async function ({ method, params }) {
+        let id = 0;
```

Comment:
Can `id` be something more descriptive? Like `requestId`?

Reply:
`requestId` makes sense to me.